optimum/exporters/tasks.py

```diff
@@ -623,9 +623,13 @@ class TasksManager:
             "audio-xvector",
             onnx="UniSpeechSATOnnxConfig",
         ),
-        "vae": supported_tasks_mapping(
+        "vae-encoder": supported_tasks_mapping(
             "semantic-segmentation",
```

Comment:
Nit: it sounds weird to have the task "semantic-segmentation" for both the encoder and decoder since they don't perform this task alone (only when they are associated with each other)

Reply:
I agree that it's not super clear, we could rename the corresponding key of [`_TASK_TO_COMMON_OUTPUTS`](https://github.com/huggingface/optimum/blob/main/optimum/exporters/onnx/base.py#L152) to something more general in a future PR (as it's not related to this one)

Reply:
Maybe create an issue to not forger about it.